src/actions/fetch/copier.ts

```diff
@@ -228,7 +162,7 @@ export function createFetchCopierAction(options: {
         values: values,
         imageName: ctx.input.imageName,
         templateDir: templateDir,
-        templateContentsDir: templateContentsDir,
+        templateContentsDir: templateLocation,
```

Comment:
This naming feels a bit confusing to me, surely the `templateLocation` is a URL, therefore not a directory? Perhaps `templateUrl` would be better?

Reply:
fairs